package.json

```diff
@@ -28,9 +28,13 @@
     "change-user": "node scripts/change-user.js",
     "change-user:admin": "npm run change-user admin@of.chapter.one",
     "change-user:owner": "npm run change-user foo@bar.com",
-    "cypress:open": "cypress open",
-    "cypress:run": "cypress run",
+    "precypress": "npm run cypress:install",
+    "cypress": "cypress",
+    "cypress:open": "npm run cypress -- open",
+    "cypress:run": "npm run cypress -- run",
     "cypress:coverage:report": "npx nyc report",
+    "cypress:install": "cypress install",
+    "cypress:install-build-tools": "sh ./cypress-install.sh",
```

Comment:
Random question, why do we need explicitly to have `npm run precypress`, while there is `npm run cypress:install`?

What the benefit over `"precypress": "cypress install"`

Reply:
Good question! `cypress` isn't a global command, it's installed locally in the chapter repo (in node modules). That means we need to invoke it via npm.

The main point is to ensure that precypress gets called (since that's responsible for installing Cypress). Hence the other scripts call the cypress script.

Final piece of the puzzle: `pre<anything>` gets called before `<anything>` (cypress, in this case)